std_msgs/Header header

uint8 status
uint32 particle_size
float32 match_ratio

uint8 NORMAL = 0
uint8 GLOBAL_LOCALIZATION = 1
uint8 EXPANSION_RESETTING = 2
